src/containers/quiz/question-answer/Answer.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  FormControlLabel,
+  FormControlLabelProps,
+  Box,
+  Checkbox,
+  Radio,
+  InputAdornment,
+  SxProps
+} from '@mui/material'
+import { ChangeEventHandler, FC } from 'react'
+import CheckIcon from '@mui/icons-material/Check'
+import CloseIcon from '@mui/icons-material/Close'
+import { spliceSx } from '~/utils/helper-functions'
+import { styles } from './Answer.styles'
+import { QuestionTypesEnum } from '~/types'
+import AppTextField from '~/components/app-text-field/AppTextField'
+import { questionType } from '~/components/question-editor/QuestionEditor.constants'
+import { useTranslation } from 'react-i18next'
+
```

Comment:
organize imports according to our convenience

Reply:
Thanks, imports were organized